io/net/bonding.py.data/bonding.yaml

```diff
@@ -0,0 +1,9 @@
+Test: !mux
+    option:
+        bonding_mode: "0 1 2 3 4 5 6"
+Parameters:
+    Iface1: "enP3p1s0f0"
+    Iface2: "enP3p1s0f1"
+    peerip: "85.85.85.85"
+    peerif2: "eth3"
```

Comment:
i tried with multiplexer input, which is unnecessarily remove bond in peer and again create bond in peer with mode 0 only( around 6+6 times unnecessarily doing)